cmd/arcstat.in

```diff
@@ -506,7 +600,8 @@ def calculate():
     global d
     global v
     global l2exist
-
+    global fieldstats
+    
```

Comment:
Was here some unneeded spaces before or are there now?  I.e. is this a fix or a new typo?